web/cobrands/fixmystreet/assets.js

```diff
@@ -319,6 +319,15 @@ OpenLayers.Layer.VectorNearest = OpenLayers.Class(OpenLayers.Layer.VectorBase, {
             return;
         }
         this.checkFeature(null, fixmystreet.get_lonlat_from_dom());
+
+        // It's possible an asset has been selected before a category (e.g. if
```

Comment:
This block is not necessary, I think. `road: true` asset layers use the `usrn:` dictionary to update key-values based on the nearest feature automatically. Instead of `attributes: location_name: site_name` for the road layers, you want `usrn: - { attribute: 'site_name', field: 'location_name' }` which should do what you're after.